lib/raygun/raygun.rb

```diff
@@ -208,6 +212,35 @@ def print_next_steps_for_custom_repo
 
     protected
 
+    def fetch_rubocop_file
+      sha = shell("git ls-remote https://github.com/#{C5_CONVENTIONS_REPO} master")
+      sha = sha.slice(0..6)
+
+      rubocop_file = "https://raw.githubusercontent.com/#{C5_CONVENTIONS_REPO}/master/rubocop/rubocop.yml"
+      begin
+        prefix = [
+          "# Sourced from #{C5_CONVENTIONS_REPO} @ #{sha}",
+          '#',
+          '# If you make changes to this file, consider opening a PR to backport them to the c5-conventions repo:',
+          "# https://github.com/#{C5_CONVENTIONS_REPO}/blob/master/rubocop/rubocop.yml",
+          '',
+          ''
+        ].join("\n")
+        rubocop_contents = URI.open(rubocop_file)
+        write_file("#{@app_dir}/.rubocop.yml", prefix + rubocop_contents.string)
```

Comment:
I like using Ruby heredoc for multiline strings like the prefix:

```suggestion
        write_file("#{@app_dir}/.rubocop.yml", <<~PREFIX + rubocop_contents.string)
          # Sourced from #{C5_CONVENTIONS_REPO} @ #{sha}"
          #
          # If you make changes to this file, consider opening a PR to backport them to the c5-conventions repo:
          # https://github.com/#{C5_CONVENTIONS_REPO}/blob/master/rubocop/rubocop.yml
          
          
        PREFIX
```

Reply:
Or taking it a step further:

```ruby
IO.write("#{@app_dir}/.rubocop.yml", <<~RUBOCOP_YML)
  # Sourced from #{C5_CONVENTIONS_REPO} @ #{sha}"
  #
  # If you make changes to this file, consider opening a PR to backport them to the c5-conventions repo:
  # https://github.com/#{C5_CONVENTIONS_REPO}/blob/master/rubocop/rubocop.yml
  
  #{rubocop_contents.string}
RUBOCOP_YML
```